js/interactive-guides/step-content.js

```diff
@@ -166,27 +166,30 @@ var stepContent = (function() {
             //console.log("displayType: ", content.displayType);
             switch (content.displayType) {
               case 'fileEditor':
-                var newEditor = editor.create(subContainer, step.name, content);
-                //console.log(newEditor);
-                contentManager.setEditor(step.name, newEditor);
+                editor.create(subContainer, step.name, content).done(function(newEditor){
+                  //console.log(newEditor);
+                  contentManager.setEditor(step.name, newEditor, displayTypeNum);
+                });                
                 break;
               case 'commandPrompt':
                 //console.log("commandPrompt detected");
                 var newCmdPrompt = cmdPrompt.create(subContainer, step.name, content);
-                contentManager.setCommandPrompt(step.name, newCmdPrompt);
+                contentManager.setCommandPrompt(step.name, newCmdPrompt, displayTypeNum);
```

Comment:
Since the cmdPrompt.js has changed to load asynchronously, need to change the create and contentManager codes to handle deferred like the other widgets.

Reply:
Fixed